data-prepper-plugins/common/src/main/java/org/opensearch/dataprepper/common/sink/DefaultSinkBuffer.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.common.sink;
+
+import java.time.Instant;
+
+public class DefaultSinkBuffer implements SinkBuffer {
```

Comment:
Curious how this will be applied to all sinks? Or is it only meant to be applied for some?